virga/_cli/templates/boilerplate/api/scripts/prestart.sh

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 
-alembic upgrade head
+# wait for postgres
+# while ! pg_isready -h "${POSTGRES_HOST:-localhost}" -U "${POSTGRES_USER}" -p 5432; do
+#     sleep 1;
+# done
+
+# run pending migrations
+# alembic upgrade head
```

Comment:
Is this supposed to be commented still?

Reply:
yea -- it gets uncommented automatically if you generate a project with `--database`, since if you don't the alembic structure isn't created for you (and the cmd would fail). the general approach throughout virga is to leave as much stuff commented in shared code as possible (and uncomment if a flag is provided) for transparency and in case you gen a project and then later decide you actually do need a feature